guides/common/attributes-titles.adoc

```diff
@@ -25,7 +25,8 @@
 :ReleaseNotesDocTitle: Release Notes
 :TuningDocTitle: Tuning Performance of {ProjectName}
 :UpdatingDocTitle: Updating {ProjectName}
-:UpgradingDocTitle: Upgrading {ProjectName} to {ProjectVersion}
+:UpgradingDocTitle: Upgrading Connected {ProjectName} to {ProjectVersion}
+:UpgradingDisconnectedDocTitle: Upgrading Disconnected {ProjectName} to {ProjectVersion}
```

Comment:
These two attributes should go under `satellite` below and you must keep the original line 28 here.